### PR TITLE
docs: update roadmap for 202211

### PR DESCRIPTION
**Pull request by ran-huang**
### What problem does this PR solve?

_**DNM until v6.4 is released**._

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -112,17 +116,17 @@ This roadmap brings you what's coming in the 1-year future, so you can see the n
...
   <tr>
     <td>Support replicating data to object storage such as S3.</td>
     <td>TiCDC supports replicating data changes to common object storage services.</td>
   </tr>
   <tr>
     <td>Data migration</td>
-    <td>TiDB Lightning supports table-level and partition-level online data import.</td>
-    <td>TiDB Lightning provides comprehensive table-level and partition-level data import capabilities.</td>
+    <td>Continuous data verification during data migration.</td>
+    <td>DM supports online data verification during  migration from MySQL compatible database to TiDB.</td>
```
Comment (hawkingrei) [inline roadmap.md:129]:
```suggestion
    <td>DM supports online data verification during migration from MySQL compatible database to TiDB.</td>
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Comment by ran-huang** [comment]
@hawkingrei Can you merge this pull request? Thank you.

**Review by hawkingrei — APPROVED** [review]

**Review by xhebox — APPROVED** [review]

**Pull request merged**